### PR TITLE
Sender name guidance updates

### PR DESCRIPTION
**Pull request by jakecosgrove**
## Description

This PR addresses the following tickets:
- [CCM-7332](https://nhsd-jira.digital.nhs.uk/browse/CCM-7332)
- [CCM-7330](https://nhsd-jira.digital.nhs.uk/browse/CCM-7330)
- [CCM-7329](https://nhsd-jira.digital.nhs.uk/browse/CCM-7329)

## Context

Users of NHS Notify need guidance to choose appropriate sender names for each channel they use to send messages with. 

Each message channel has their own considerations and processes to use in order to create an approved sender name.

This content provided additional guidance for users to choose their own sender names.

## Type of changes

- [x] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md [hunk 1]
```diff
@@ -16,32 +16,170 @@ You can change the:
 
 - [NHS App sender name](#nhs-app-sender-name)<!-- markdownlint-disable-line -->
 - [text message sender name](#text-message-sender-name)
+- [email sender name and from address](#)
```
Comment (alexcolbeck1) [inline docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md:19]: This link seems to be missing an anchor link.

File: docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md [hunk 2]
```diff
@@ -16,32 +16,170 @@ You can change the:
...
 - [reply-to email address](#reply-to-email-address)
 
-## Return addresses
+## Return addresses for letters
 
 You cannot change the return addresses of letters. These are set by NHS Notify’s suppliers and are printed on the envelopes.
 
+If you want users to reply to you letters, include your organisation or service's address in the content of your [letter template]({% link pages/using-nhs-notify/letter-templates.md %}).
```
Comment (alexcolbeck1) [inline docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md:26]: Assumed spelling error:

> If you want users to reply to **your** letters...

File: docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md [hunk 3]
```diff
@@ -16,32 +16,170 @@ You can change the:
...
 
+Your NHS App sender name is what appears to your recipients when you send them messages to the NHS App.
+
 The NHS App uses your service or organisation’s ODS code to set up your NHS App message sender name.
 
-NHS Notify will ask you for your ODS code during onboarding.
+NHS Notify will ask you for your ODS code during onboarding. [Find your ODS code with the ODS Portal](https://odsportal.digital.nhs.uk/).
 
-Find your ODS code with the [ODS Portal](https://odsportal.digital.nhs.uk/).
+If your organisation or service does not have an ODS code, you can [request a code or update your organisation's details](https://digital.nhs.uk/services/organisation-data-service/request-a-new-code-or-update-organisation-details).
```
Comment (alexcolbeck1) [inline docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md:36]: Links to external sites should open in a new tab really. Apparently later versions of Markdown support `target="_blank"`:

`[link](URL){:target="_blank"}`

File: docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md [hunk 4]
```diff
@@ -16,32 +16,170 @@ You can change the:
...
+- winner'
+
+%}
+
+## Email sender name and 'from' address
+
+The email sender name is what appears in your recipient's inbox when you send them an email. It appears alongside the email address your email was sent from, also known as the 'from' address.
+
+Before you can send emails, you must choose and provide your email sender name and 'from' address during onboarding.
+
```
Comment (alexcolbeck1) [inline docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md:135]: Just an idea, could you have a screenshot of an example showing the sender name and from address in an email sent by NHS Notify?

**Review by alexcolbeck1 — DISMISSED** [review]

File: docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md (hunk 3 above)
Comment (jakecosgrove) [reply docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md:36]: Good tip, will do a PR with these added everywhere now!

File: docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md (hunk 4 above)
Comment (jakecosgrove) [reply docs/pages/using-nhs-notify/tell-recipients-who-your-messages-are-from.md:135]: It's a good idea, will look into it!

**Review by mark-r-bjss — APPROVED** [review]

**Pull request merged**